website/pages/docs/plugins/sources/alicloud/_configuration.mdx

```diff
@@ -4,6 +4,8 @@ spec:
   name: "alicloud"
   path: "cloudquery/alicloud"
   version: "VERSION_SOURCE_ALICLOUD"
+  # adjust this value up or down to change the number of tables that are resolved in parallel
```

Comment:
```suggestion
  # adjust this value up or down to change the number of requests that are made in parallel
```

I think the current phrasing might leave the impression that concurrency is a 1:1 mapping to number of tables synced in parallel, so we may want to rephrase it slightly